### ISSUE TITLE
namyats delayed a round?

### ISSUE THREAD

**Issue by Muppet1** [EdwardPiwowar/BBA#774]
![image](https://github.com/user-attachments/assets/1bbe7338-b6d3-4e1e-9339-f4433eb8aa36)

So here I noticed 4d was artificial so I thought I would bid it to see what it was and the answer was really good hearts which is completely crazy.

**Comment by EdwardPiwowar** [comment]
For me 4D is interpreted as artificial (and it's not really defined):
![image](https://github.com/user-attachments/assets/cace1fe3-fea1-43b3-bccf-77eae9685824)
Paste BBA number.

**Comment by Muppet1** [comment]
yes sorry forgot number added it now.  yes artificial until you actually make the bid then it changes to really good hearts.  Hmm well it did the first time can't actually get it to show that now I'll keep trying

**Comment by EdwardPiwowar** [comment]
PC requires removal of bugs introduced by recent changes.

**Comment by Muppet1** [comment]
ok I'll leave PC alone for a bit then was going to make PC net next but I'll do something else first.
